Library/Formula/rethinkdb.rb

```diff
@@ -32,6 +32,81 @@ def install
     system "make", "install-osx"
 
     (var/"log/rethinkdb").mkpath
+
+    (buildpath+"rethinkdb.conf").write rethinkdb_conf
+    etc.install "rethinkdb.conf"
```

Comment:
Does `rethinkdb` have a sample configuration we can copy/modify rather than storing it all in the formula?